errors/export_test.go

```diff
@@ -1,6 +0,0 @@
-// Copyright 2014 Canonical Ltd.
```

Comment:
The `errors` package is deprecated. It should not have any relevance. See https://github.com/giantswarm/errgo/pull/1/files#diff-67bcaf0cce945ce4f52191db72018763L11.